### PR TITLE
Expose permute inputs

### PR DESCRIPTION
**Pull request by makslevental**

### CODE REVIEW

**Comment by makslevental** [comment]
cc @Krovatkin @eellison

**Review by eellison — APPROVED** [review]

File: test/test_jit.py [hunk 1]
```diff
@@ -1688,6 +1688,20 @@ def doit(x, y):
         for node in g.nodes():
             self.assertTrue(g2.findNode(node.kind()) is not None)
 
+    def test_permute_inputs_binding(self):
+        g = torch._C.Graph()
```
Comment (eellison) [inline test/test_jit.py:1692]: looks like this is failing - maybe try by using a graph from a function ?

```
@torch.jit.script
def foo(i, j, k):
     pass

g = foo.graph
```

Comment (makslevental) [reply test/test_jit.py]: my bad i should've test this locally before i push. thanks for the trip

**Pull request closed without merging**